v2/cmd/asoctl/cmd/root.go

```diff
@@ -56,8 +60,14 @@ func newRootCommand() (*cobra.Command, error) {
 	}
 
 	rootCmd.PersistentPreRun = func(cmd *cobra.Command, args []string) {
+		// Configure logging; --verbose overrides --quiet
```

Comment:
Could instead error if both are set?

Reply:
Now logging an error, but don't want to abort the run for a trivial matter.